app/src/main/java/com/framgia/englishconversation/data/source/remote/timeline/TimelineRemoteDataSource.java

```diff
@@ -3,6 +3,7 @@
 import android.support.annotation.NonNull;
 
 import com.framgia.englishconversation.data.model.Status;
+import com.framgia.englishconversation.data.model.StatusModel;
```

Comment:
ERROR: Unused import: com.framgia.englishconversation.data.model.StatusModel. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports